src/deepsparse/loggers/prometheus_logger.py

```diff
@@ -130,8 +130,6 @@ def _setup_client(self):
     def _validate(self, value: Any) -> Any:
         # make sure we are passing a value that is
         # a valid metric by prometheus client's standards
-        if hasattr(value, "is_integer"):
```

Comment:
any reason not to keep this but add a check for numpy?

Reply:
I explain this in the PR:
**We should reserve PrometheusLogger for metrics (floats/ints) only, as this is an implicit assumption when one uses Prometheus. If the user wishes to log more complex data structures, they are free to use CustomLogger or one of the future Loggers we will introduce soon.**